library/Falcon/Connector/WordPress.php

```diff
@@ -505,13 +505,16 @@ public function get_comment_subscribers( WP_Comment $comment ) {
 			);
 		}
 
-
 		$users = get_users( $query );
 
+		// Also get the original post author, they should always be notified.
+		$author_id = get_post_field( 'post_author', $comment->comment_post_ID );
```

Comment:
This is a pretty uncommon function, and also isn't quite what we want here I think. `get_post_field` by default will sanitize the return value for display, which might not do anything, but if eg someone added comma formatting for IDs maybe it would.

It'd be more obvious and less potentially error prone to do something like:
```suggestion
		$original_post = get_post( $comment->comment_post_ID );
		if ( $original_post ) {
			$author_id = $original_post->post_author;
			$author_data = $author_id ? get_user_by( 'id', $author_id ) : null;
		}
```

(Adding error checking just in case.)